payment/src/main/java/com/kyriba/conference/payment/PaymentApplication.java

```diff
@@ -0,0 +1,35 @@
+package com.kyriba.conference.payment;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+import java.time.LocalDateTime;
+
+
+/**
+ * @author Igor Lizura
+ */
+@EnableJpaRepositories
+@SpringBootApplication
+@EnableSwagger2
+public class PaymentApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(PaymentApplication.class, args);
+	}
+
+	@Bean
+	public Docket paymentApi() {
+		return new Docket(DocumentationType.SWAGGER_2)
+				.directModelSubstitute(LocalDateTime.class, String.class)
```

Comment:
Provide general API info and apply filter on URL.

Reply:
done